scylla/src/main/java/site/ycsb/db/scylla/ScyllaCQLClient.java

```diff
@@ -87,7 +87,7 @@
   public static final String SCYLLA_LWT = "scylla.lwt";
 
   public static final String TOKEN_AWARE = "scylla.tokenaware";
-  public static final String TOKEN_AWARE_DEFAULT = "false";
+  public static final String TOKEN_AWARE_DEFAULT = "true";
```

Comment:
this shouldn't have any impact, if there is just single DC
look at
https://docs.datastax.com/en/developer/java-driver/3.10/manual/load_balancing/
```
If you don’t explicitly configure the policy, you get the default, which is a datacenter-aware, token-aware policy:

new TokenAwarePolicy(DCAwareRoundRobinPolicy.builder().build());
```

Reply:
so
TOKEN_AWARE_LOCAL_DC 
is the only param that is needed, in case there are multi DCs

Reply:
oh. I didn't know that it works by default.

Reply:
@tarzanek I have removed this flag then "scylla.tokenaware" as it is does not make sense having it.